ui/app/components/ui/dropdown/dropdown.js

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const Dropdown = ({ className, disabled, name, onChange, options, required, selectedOption, style }) => {
+  const _onChange = useCallback(
+    (event) => {
+      event.preventDefault()
+      event.stopPropagation()
+      onChange(event.target.value)
+    },
+    [onChange],
+  )
+
+  return (
+    <select
+      className={classnames('dropdown', className)}
+      disabled={disabled}
+      name={name}
+      onChange={_onChange}
+      required={required}
```

Comment:
I don't think this makes sense outside of the context of a form, and this component isn't really meant to be used in a form. I'll remove it for now.

Reply:
It has been removed.